### PR TITLE
completed task

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/layout_catalog/)
- [TEST REPORT LINK](https://andriy-shymkiv.github.io/layout_catalog/report/html_report/)

@Olexandr6 @mykola-koval some tests are failing and i don't know why

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,388 @@
...
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
```
Comment (andreyolin) [inline src/index.html:89]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,388 @@
...
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
```
Comment (andreyolin) [inline src/index.html:94]: you can use p here

File: src/index.html [hunk 3]
```diff
@@ -1,12 +1,388 @@
...
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
```
Comment (andreyolin) [inline src/index.html:128]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 4]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:168]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,388 @@
...
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
```
Comment (andreyolin) [inline src/index.html:213]: you can use p here

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,388 @@
...
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
```
Comment (andreyolin) [inline src/index.html:248]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 7]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:288]: </div>

           <div class="card__review">Reviews: 5</div>

need space here i think.
Also you can use p here

File: src/index.html [hunk 8]
```diff
@@ -1,12 +1,388 @@
...
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
```
Comment (andreyolin) [inline src/index.html:253]: you can use p here

File: src/index.html [hunk 9]
```diff
@@ -1,12 +1,388 @@
...
+        </div>
+
+        <a
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover"
+          class="card__buy-button"
```
Comment (andreyolin) [inline src/index.html:259]: add data-qa="card-hover" (not just hover) to the link Buy inside the first card

File: src/index.html [hunk 10]
```diff
@@ -1,12 +1,388 @@
...
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
```
Comment (andreyolin) [inline src/index.html:293]: you can use p here

File: src/index.html [hunk 11]
```diff
@@ -1,12 +1,388 @@
...
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
```
Comment (andreyolin) [inline src/index.html:328]: </div>

       <div class="card__review">Reviews: 5</div>
need space here i think.
Also you can use p here

File: src/index.html [hunk 12]
```diff
@@ -1,12 +1,388 @@
...
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
```
Comment (andreyolin) [inline src/index.html:333]: you can use p here

File: src/index.html [hunk 13]
```diff
@@ -1,12 +1,388 @@
...
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review">Reviews: 5</div>
```
Comment (andreyolin) [inline src/index.html:368]: </div>

       <div class="card__review">Reviews: 5</div>
need space here i think.
Also you can use p here

File: src/index.html [hunk 14]
```diff
@@ -1,12 +1,388 @@
...
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price</span>
+          <span class="card__price-amount">$2,199</span>
```
Comment (andreyolin) [inline src/index.html:373]: you can use p here

File: src/styles/blocks/stars.scss [hunk 15]
```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
```
Comment (andreyolin) [inline src/styles/blocks/stars.scss:5]: the last star should not have margin

File: src/index.html [hunk 16]
```diff
@@ -1,12 +1,388 @@
...
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+
+      </nav>
```
Comment (andreyolin) [inline src/index.html:62]: delete space

File: src/index.html [hunk 17]
```diff
@@ -1,12 +1,388 @@
...
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover" class="card__buy-button"
+        >
+          Buy
+        </a>
+
```
Comment (andreyolin) [inline src/index.html:103]: delete space

File: src/index.html [hunk 18]
```diff
@@ -1,12 +1,388 @@
...
+      </div>
```
Comment (andreyolin) [inline src/index.html:104]: delete space

File: src/index.html [hunk 19]
```diff
@@ -1,12 +1,388 @@
...
+
+        <a
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover"
+          class="card__buy-button"
+        >
+          Buy
+        </a>
+
+      </div>
```
Comment (andreyolin) [inline src/index.html:144]: delete space

File: src/index.html [hunk 20]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:184]: delete space

File: src/index.html [hunk 21]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:224]: delete space

File: src/index.html [hunk 22]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:264]: delete space

File: src/index.html [hunk 23]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:304]: delete space

File: src/index.html [hunk 24]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:344]: delete space

File: src/index.html [hunk 25]
```diff
@@ -1,12 +1,388 @@
...
+
+      <div>
+        <a href="#home">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
```
Comment (andreyolin) [inline src/index.html:22]: delete space

File: src/index.html (hunk 16 above)
Comment (andreyolin) [inline src/index.html:62]: delete space

File: src/index.html [hunk 26]
```diff
@@ -1,12 +1,388 @@
...
+        </ul>
+
+      </nav>
+
+    </header>
+    <main class="content">
+
+      <div class="card" data-qa="card">
+
+        <img
```
Comment (andreyolin) [inline src/index.html:69]: <main class="content">
      <div class="card" data-qa="card">
        <img

delete spaces

File: src/index.html [hunk 27]
```diff
@@ -1,12 +1,388 @@
...
+          class="card__buy-button"
+        >
+          Buy
+        </a>
+
+      </div>
+
+      <div class="card">
+
+        <img
```
Comment (andreyolin) [inline src/index.html:148]: delete space

File: src/index.html [hunk 28]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:188]: delete space

File: src/index.html [hunk 29]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:228]: delete space

File: src/index.html [hunk 30]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:308]: delete space

File: src/index.html [hunk 31]
```diff
@@ -1,12 +1,388 @@
...
```
Comment (andreyolin) [inline src/index.html:348]: delete space

File: src/styles/utils/_reset.scss [hunk 32]
```diff
@@ -0,0 +1,20 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
```
Comment (andreyolin) [inline src/styles/utils/_reset.scss:5]: you have some problems with fonts. For example:

.card__code should have font-weight 400
.card__price-amount should have color #060b35
etc.

File: src/index.html [hunk 33]
```diff
@@ -1,12 +1,388 @@
...
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div>
+        <a href="#home">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div>
```
Comment (andreyolin) [inline src/index.html:18]: do you need div here? 
As i can see this div have size 40*43.6, but logo is 40*40.
Thats why logo is not at the center of header

**Review by andreyolin — APPROVED** [review]

File: src/index.html [hunk 34]
```diff
@@ -1,12 +1,392 @@
...
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="moyo">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
```
Comment (Olexandr6) [inline src/index.html:26]:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
```

File: src/index.html [hunk 35]
```diff
@@ -1,12 +1,392 @@
...
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+
+      </nav>
```
Comment (Olexandr6) [inline src/index.html:66]: Don't add redudundant empty lines!

Comment (Olexandr6) [reply src/index.html:66]: Fix this in all places

File: src/index.html [hunk 36]
```diff
@@ -1,12 +1,392 @@
...
+
+    </header>
+    <main class="content">
```
Comment (Olexandr6) [inline src/index.html:69]:
```suggestion
    </header>
    
    <main class="content">
```

File: src/index.html [hunk 37]
```diff
@@ -1,12 +1,392 @@
...
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price</p>
```
Comment (Olexandr6) [inline src/index.html:97]: It's the difference I've noticed from your test report
```suggestion
          <p class="card__price-title">Price:</p>
```

File: src/index.html [hunk 38]
```diff
@@ -1,12 +1,392 @@
...
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover" class="card__buy-button"
```
Comment (Olexandr6) [inline src/index.html:103]: image

Comment (Olexandr6) [inline src/index.html:103]: How do we format attributes in case there are 3 ones or more?

File: src/index.html (hunk 18 above)
Comment (Olexandr6) [reply src/index.html:108]: +1

File: src/index.html [hunk 39]
```diff
@@ -1,12 +1,392 @@
...
+          href="https://www.apple.com/ua/imac-24/"
+          data-qa="hover"
+          class="card__buy-button"
+        >
+          Buy
+        </a>
+
+      </div>
+
+    </main>
```
Comment (Olexandr6) [inline src/index.html:390]:
```suggestion
        </a>
      </div>
    </main>
```

File: src/styles/blocks/card.scss [hunk 40]
```diff
@@ -0,0 +1,125 @@
...
+    line-height: 16px;
+  }
+
+  &__buy-button:hover {
+    background-color: #fff;
+    border-color: #00acdc;
+    color: #00acdc;
+  }
+
+}
```
Comment (Olexandr6) [inline src/styles/blocks/card.scss:125]:
```suggestion
  }
}
```

File: src/styles/blocks/card.scss [hunk 41]
```diff
@@ -0,0 +1,125 @@
...
+
+    text-decoration: none;
+    text-transform: uppercase;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  &__buy-button:hover {
```
Comment (Olexandr6) [inline src/styles/blocks/card.scss:119]: [Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point

File: src/styles/blocks/navigation.scss [hunk 42]
```diff
@@ -0,0 +1,73 @@
...
+    height: 60px;
+
+    color: #060b35;
+
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__link:hover {
+    color: #00acdc;
```
Comment (Olexandr6) [inline src/styles/blocks/navigation.scss:52]: [Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point

File: src/styles/blocks/navigation.scss [hunk 43]
```diff
@@ -0,0 +1,73 @@
...
+  }
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
```
Comment (Olexandr6) [inline src/styles/blocks/navigation.scss:61]: [Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point

File: src/styles/blocks/stars.scss (hunk 15 above)
Comment (Olexandr6) [reply src/styles/blocks/stars.scss:5]: @andreyolin Good point!
Fix this

File: src/styles/main.scss [hunk 44]
```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import"./utils/variables";
+@import "./blocks/navigation.scss";
+@import "./utils/fonts.scss";
+@import "./utils/reset.scss";
+@import "./blocks/content.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
```
Comment (Olexandr6) [inline src/styles/main.scss:7]:
```suggestion
@import"./utils/variables";
@import "./utils/fonts.scss";
@import "./utils/reset.scss";

@import "./blocks/navigation.scss";
@import "./blocks/content.scss";
@import "./blocks/card.scss";
@import "./blocks/stars.scss";
```

File: src/styles/utils/_reset.scss [hunk 45]
```diff
@@ -0,0 +1,20 @@
...
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+
+  line-height: 14px;
+}
+
+* {
+  box-sizing: border-box;
+}
```
Comment (Olexandr6) [inline src/styles/utils/_reset.scss:11]: It's not `reset` styles, rather `normalize`. Move these styles to normalize.scss

File: src/styles/blocks/navigation.scss [hunk 46]
```diff
@@ -0,0 +1,73 @@
...
+  }
+
+  &__link:hover {
+    color: #00acdc;
+  }
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```
Comment (Olexandr6) [inline src/styles/blocks/navigation.scss:58]: You use this color in the several places, so make sense to create a variable for it

File: src/styles/blocks/navigation.scss [hunk 47]
```diff
@@ -0,0 +1,73 @@
...
+    margin-left: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+
+    height: 60px;
+
+    color: #060b35;
```
Comment (Olexandr6) [inline src/styles/blocks/navigation.scss:45]: [Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 11th point

Comment (Olexandr6) [reply src/styles/blocks/navigation.scss:45]: Review all colors and check for what you should create variables

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Looks like you have reviewed the Checklist in a not-good way

**Pull request closed without merging**